server/views/pages/barcode/create-new-contact.njk

```diff
@@ -53,7 +53,8 @@
               label: {
                 text: 'Select name of prison'
               },
-              items: prisonRegister
+              items: prisonRegister,
+              errorMessage: errors | findError('prisonId')
```

Comment:
This is the fix for SLM-92 (error highlight and message not appearing on prison name field) - I forgot to add it when I originally wrote the markup for the page 🙄 
We like easy fixes like this 👍 😁